projects/hslayers/src/components/layermanager/layermanager.service.ts

```diff
@@ -365,6 +365,10 @@ export class HsLayerManagerService {
     if (this.HsLayerUtilsService.isLayerArcgis(layer)) {
       return 'ARCGIS';
     }
+
+    if (this.HsLayerUtilsService.isLayerIDW(layer)) {
+      return 'IDW';
```

Comment:
I am not sure now about the actual implementation.. Is the layer always interpolated via the IDW algorithm? If so, then the return 'IDW' is perfectly fine, if other interpolation methods could be used then it would be misleading.

Reply:
Currently only IDW, as far as I know.

Reply:
Acknowledged.